Example/ReusableDemo/CollectionViewCells/MyXIBIndexSquaceCell.swift

```diff
@@ -20,7 +20,7 @@ import Reusable
  * Which in fact is just a convenience protocol that combines
  * both `NibLoadable` + `Reusable` protocols.
  */
-final class MyXIBIndexSquaceCell: UICollectionViewCell, NibReusable {
+final class MyXIBIndexSquaceCell: UICollectionViewCell, Reusable, NibLoadable {
```

Comment:
Now that we've decided to keep `NibReusable` as a typealias, maybe keep the usage of `NibReusable` in the example project here, especially since it's already explained in the comment that it's in fact just a convenience name combining both.

btw, you might also want to check the doc-comments throughout the code and sample project where we describe `NibReusable` as "just a convenience protocol that combines both `NibLoadable` + `Reusable` protocols" and change it to say it's a "convenience typealias combining `NibLoadable & Reusable`" instead.